server/app/utils.py

```diff
@@ -568,20 +578,17 @@ def make_zip_filename(user, now):
     return filename+'.zip'
 
 
-def subms_to_gcs(SearchAPI, subm, Submission, user, data, datetime,
-                start_cursor=None):
+def subms_to_gcs(SearchAPI, subm, Submission, filename, data):
     """Writes all submissions for a given search query to a GCS zip file."""
-    zipfile_str, zipfile = start_zip()
-    next_cursor, has_more = None, True
-    while has_more:
-        query = SearchAPI.querify(data['query'])
-        results, next_cursor, has_more = query.fetch_page(
-            constants.BATCH_SIZE, start_cursor=next_cursor)
-        for result in results:
-            zipfile = add_subm_to_zip(subm, Submission, zipfile, result)
-    zip_contents = finish_zip(zipfile_str, zipfile)
-    zip_filename = make_zip_filename(user, datetime)
-    create_gcs_file(zip_filename, zip_contents, 'application/zip')
+    query = SearchAPI.querify(data['query'])
+    gcs_file = gcs.open(filename, 'w',
+        content_type='application/zip',
+        options={'x-goog-acl': 'project-private'})
```

Comment:
Sidenote: It'd be nice to open have a proxy endpoint so we don't have to go to the console everytime - but that's not really needed. 

We could do auth by storing the course ID as a `x-goog-meta-course` object and then have api/v1/course/files/filename. It's not a huge priority right now though.